_src-js/lib/bound-properties.js

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import ReactDOM from 'react-dom'
+import videojs from 'video.js';
+import shortid from 'shortid';
+
+const Form = React.createClass({
+  getInitialState() {
+    return {
```

Comment:
Not a big deal, just curious why you decided not to go with ES6 syntax for everything?

```javascript
class Form extends React.Component {
  constructor(props) {
    super(props);
    this.state = { ... };
  }

  ...
}
```